### ISSUE TITLE
GraphQL: make sure that interface implementations all accept the same arguments for each field

### ISSUE THREAD

**Issue by lutter** [graphprotocol/graph-node#1349]
This came up in the review of PR #1341

**Comment by azf20** [comment]
@lutter @dotansimha do you know if this issues still applies?

**Comment by lutter** [comment]
Yes, I don't think anything has changed there. This would be a validation that should go into `graph-cli` and into the validation of the GraphQL schema when a subgraph is deployed in `graph-node`
